cypress/integration/tip.test.ts

```diff
@@ -0,0 +1,100 @@
+import { Currency, CurrencyAmount, JSBI, Token, TokenAmount, Trade, WETH } from '@uniswap/sdk'
+import { getFeeAmount } from '../../src/custom/utils/fee'
+
+const stringToCurrency = (amount: string, currency: Currency) =>
+  currency instanceof Token ? new TokenAmount(currency, JSBI.BigInt(amount)) : CurrencyAmount.ether(JSBI.BigInt(amount))
+
+// Assumptions:
+// MAINNET
+// SELL: ETH
+// BUY: DAI
+const INPUT_TOKEN = WETH[1].address
+const OUTPUT_TOKEN = '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984'
+const DAI = '0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735'
+
+const INPUT_AMOUNT = '50'
+
+const FEE_QUERY = `https://protocol-mainnet.dev.gnosisdev.com/api/v1/tokens/${OUTPUT_TOKEN}/fee`
+
+const inputIsETH = INPUT_TOKEN === WETH[1].address
+const outputIsETH = OUTPUT_TOKEN === WETH[1].address
```

Comment:
If both INPUT_TOKEN and OUTPUT_TOKEN are constants, why do you need the conditional?

Reply:
I was testing this earlier for sth else I'll remove